README.md

```diff
@@ -1,3 +1,5 @@
 # HYF Coach Bios
 
 A little hello and introduction from all of your coaches:
+
+* Joel[JoeCamacho.md]
```

Comment:
@JoeCamacho broken syntax here for the link, it should be `[Joel](JoeCamacho.md)`